### ISSUE TITLE
Can't run a model not at the root of a repo

### ISSUE THREAD

**Issue by dgryski** [salesforce/runway-browser#9]
My model checking playground is `github.com/dgryski/modelchecking`.  I have a simulation at `github.com/dgryski/modelchecking/runway/river` that I can't figure out the URL structure to be able to view it on runway.systems.

(I exported it to its own repo so that `http://runway.systems/?model=github.com/dgryski/runway-model-river` works)

**Comment by ongardie-sfdc** [comment]
Seems like this URL works https://runway.systems/?model=github.com/dgryski/modelchecking/master/runway/river

This should be documented somewhere, of course.

And is there some de facto syntax for a Git repo easy to put in a URL, usually but not always GitHub, possibly at a particular branch/tag/sha1, possibly with a path within that repo? I made something up here, but it's probably not the right thing.

PS. I know you've left a few comments around. Please keep doing that :). Sorry I'm slow, but I'll get to the rest soon.

**Comment by dgryski** [comment]
Yes, that does work.  I had tried a number of variations including urls with `tree` and `blob`.  I think this is just a docs issue then.

There isn't a standard URL structure.  runway-browser would need to know about the popular hosting providers (github, bitbucket, gitlab), plus some way to handle custom domains should be more than sufficient.

**Comment by ongardie-sfdc** [comment]
Maybe npm, godoc, sourcegraph, etc have come up with something sane?

/cc @sqs

**Comment by sqs** [comment]
[Sourcegraph](https://sourcegraph.com) uses the [Go remote import path spec](https://golang.org/cmd/go/#hdr-Remote_import_paths) to map from paths to `(git URL, commit, subdir)`. It's not universal, and it has ambiguity, so it's not a solution. To be truly correct, I think you'd need to have people specify all of those elements separately, plus the VCS type (git/hg/etc.).

**Comment by ongardie-sfdc** [comment]
@sqs thanks for the pointer. I agree that this doesn't look ideal either. Hmm...
